<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>135</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add New Line</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPlainTextEdit" name="newLine">
     <property name="minimumSize">
      <size>
       <width>360</width>
       <height>28</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>28</height>
      </size>
     </property>
     <property name="toolTip">
      <string>new line</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="newGloss">
     <property name="minimumSize">
      <size>
       <width>360</width>
       <height>28</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>28</height>
      </size>
     </property>
     <property name="toolTip">
      <string>new gloss (optional if line is unparseable)</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="leftMargin">
      <number>40</number>
     </property>
     <item>
      <widget class="QCheckBox" name="parsingBox">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="toolTip">
        <string>check this box if the line is not a parseable
line in the target language (e.g., it is spoken
in the contact language, it is an interjectiion,
etc.). For interlocutors other than the main 
narrator, a change of speaker, etc., add the
initials plus a colon and a space to the gloss.</string>
       </property>
       <property name="text">
        <string>unparseable?</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="minimumSize">
        <size>
         <width>164</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>164</width>
         <height>32</height>
        </size>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
